ingestion/functions/parsing/peru/requirements.txt

```diff
@@ -0,0 +1 @@
+pandas==1.1.0
```

Comment:
you don't need pandas to read a one line CSV, python has that built-in, please check the other tests for example.

Reply:
agreed, i can use the csv module, but this requires extra lines to map column names to indices. Is this extra complication preferable to the additional memory required to load the fairly bulky pandas module? Either works for me

Reply:
If all you want is to access the columsn by their names you can use a csv.DictReader
https://docs.python.org/3/library/csv.html#csv.DictReader